doc/api/errors.md

```diff
@@ -242,6 +250,15 @@ function MyError() {
 new MyError().stack;
 ```
 
+We recommend using classes that extend `Error` for custom errors:
```

Comment:
Error.captureStackTrace may even be used in combination with custom errors. It's usage is mainly to avoid specific stack frames. It's therefore not really an alternative to classes.

Reply:
Oh right, that makes sense. I've only thought about hiding constructors, which is not necessary when you subclass `Error`. I'll just get rid of the whole 'We recommend...' part including the code example.